Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -194,3 +229,17 @@ private func pledgeCTAButtonStyle(
       |> (UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode) .~ lineBreakMode
   }
 }
+
+private let pledgeRetryButtonStyle: ButtonStyle = { button in
```

Comment:
Think we need to take another look at this button style, the highlight state causes the text to disappear completely.

Also, the image doesn't have a down state, we could give it a tint for that.

We should use the `mixDarker` and `mixLighter` extensions for the highlight and disabled states - see `ButtonStyles.swift`.

<img src="https://user-images.githubusercontent.com/3735375/63128712-757bd880-bf6a-11e9-820d-dad4b1dbf8e1.gif" width="50%"/>

Reply:
Dynamic type is also a little weird with this button:

![image](https://user-images.githubusercontent.com/3735375/63128773-abb95800-bf6a-11e9-8d4c-43c0882d9ba8.png)

Reply:
For the `.highlighted` state I created an UIIMage extension and how we are able to apply alphas to images, also changed the color of the text for that state to `gray` (and used mixedLighter).

For the dynamic type I set the correct insets for the text (instead of image) and it seems to solve this issue.